packages/bot-skeleton/src/scratch/blocks/Binary/Trade Definition/multiplier_stop_loss.js

```diff
@@ -37,19 +38,22 @@ Blockly.Blocks.multiplier_stop_loss = {
             category: Blockly.Categories.Trade_Definition,
         };
     },
+    customContextMenu(menu) {
+        modifyContextMenu(menu);
+    },
     meta() {
         return {
             display_name: localize('Stop loss (Multiplier)'),
             description,
         };
     },
     onchange(event) {
-        if (!this.workspace || this.isInFlyout || this.workspace.isDragging()) {
+        if (!this.workspace || Blockly.derivWorkspace.isFlyout_ || this.workspace.isDragging()) {
             return;
         }
         if (
             (event.type === Blockly.Events.BLOCK_CREATE && event.ids.includes(this.id)) ||
-            event.type === Blockly.Events.END_DRAG
+            (event.type === Blockly.Events.BLOCK_DRAG && !event.isStart)
```

Comment:
```suggestion
            (event.type === Blockly.Events.BLOCK_DRAG && event.isEnd)
```
maybe anything else that marks the end of of an event

Reply:
just checked the events previously they were providing end drag. But now they are not providing it so had to combine drag and !isStart 

these are the events they are providing.

![image](https://github.com/user-attachments/assets/208c598d-8b0a-4bfa-ac3e-ab048a980ff0)